pom.xml

```diff
@@ -555,8 +555,10 @@
         <version.sun.jaxb>${version.org.glassfish.jaxb}</version.sun.jaxb>
         <version.wsdl4j>1.6.3</version.wsdl4j>
         <version.xml-resolver>1.2</version.xml-resolver>
-        <version.xom>1.3.8</version.xom>
```

Comment:
`version.xom` was degraded here, is there any reason behind it?

Reply:
Sorry, I faced conflicts and at that time it was degraded. I have updated the version